harvest/cli.py

```diff
@@ -90,16 +90,17 @@ def _init_arguments(self):
             'filepath',
             help=(
                 'the relative path to a file in a git repository '
-                'that you wish to retrieve'
-            )
+                'that you wish to retrieve.'
+            ),
+            nargs='+'
         )
         self.add_argument(
             '--end',
             help=(
                 'the end of date range for the file you wish to retrieve - '
                 'defaults to the current date'
             ),
-            metavar='YYYYMMDD',
+            metavar='YYYY-MM-DD',
```

Comment:
Sure I mentioned that this is an unnecessary breaking change. Suggest we have a function that takes a date in either YYYYMMDD or YYYY-MM-DD format and returns the datetime object for it. That will preserve current behaviour for anyone who's got a script using the tool and support both ISO date formats.

Reply:
Nice one